__DEFINES/callback.dm

```diff
@@ -0,0 +1,4 @@
+// Set a callback's thing_to_call to GLOBAL_PROC in order to call a global proc.
+#define GLOBAL_PROC "fuck you duny"
```

Comment:
huh

Reply:
It can be anything as long as it's not null or empty